backend/backend/settings/base.py

```diff
@@ -361,7 +361,7 @@ def get_required_setting(
 
 WSGI_APPLICATION = "backend.wsgi.application"
 
-
+ATOMIC_REQUESTS = os.environ.get("DJANGO_ATOMIC_REQUESTS", False)
```

Comment:
NIT: @johnyrahul this could be defaulted to `True` to not affect the current OSS until we test and confirm this change.

Reply:
@chandrasekharan-zipstack no . Lets keep it False only

Reply:
Intention is to disable this for ever. The ENV is optional thing. So even if we miss to add the env we should not enable this